scripts/drawer/lobby.js

```diff
@@ -316,7 +320,10 @@ class Lobby {
 		// Only enable settings if you're lobby owner
 		this.panels.detailsSettingsButton.enabled = UserAPI.GetXUID() === owner;
 
-		$.GetContextPanel().SetDialogVariable('lobbyTitle', `${FriendsAPI.GetNameForXUID(owner)}'s Lobby`);
+		$.GetContextPanel().SetDialogVariable(
+			'lobbyTitle',
+			$.Localize('#Lobby_Owner').replace('%owner%', FriendsAPI.GetNameForXUID(ownerSteamID))
```

Comment:
Should be `owner` and not `ownerSteamID`, found by Slidy in Red discord

Reply:
Woops - should be fixed now